src/features/services/screens/ServicesScreen.tsx

```diff
@@ -163,29 +165,24 @@ export const ServicesScreen = () => {
         unReadCount: unreadEmailsQuery.data
           ? unreadEmailsQuery.data.unreadEmails
           : 0,
-        onPress: () => {
-          queryClient
-            .fetchQuery(WEBMAIL_LINK_QUERY_KEY, GetWebmailLink, {
-              staleTime: 55 * 1000, // 55 seconds
-              cacheTime: 55 * 1000, // 55 seconds
-            })
-            .then(res => Linking.openURL(res.url ?? ''));
-        },
+        onPress: () => openInAppLink(urlEmailsQuery.data?.url ?? ''),
```

Comment:
we have to retrieve the link on-tap since it includes an access token that would expire if one stays too long on the page.